config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -83,7 +83,7 @@
 # Specify whether Query Logs will format tags using the SQLCommenter format
 # (https://open-telemetry.github.io/opentelemetry-sqlcommenter/), or using the legacy format.
 # Options are `:legacy` and `:sqlcommenter`.
-# Rails.application.config.active_record.query_log_tags_format = :sqlcommenter
+Rails.application.config.active_record.query_log_tags_format = :sqlcommenter
```

Comment:
This changes the default format of the feature -- but we are not using the feature. Would potentially impact instances that were using this feature ... but presumably anyone changing config to turn this on is sophisticated enough to understand the formats available.